_posts/2015-04-01-tackling-strongly-correlated-quantum-systems.md

```diff
@@ -0,0 +1,118 @@
+In many cases, these units might be referred to as particles or more generally as “quantum degrees of freedom.”
+The most exciting physics arises when these units are strongly correlated: the behavior of each one depends on the
+system as a whole; they cannot be taken and studied independently.
+Such systems arise naturally in many areas of fundamental physics, ranging from condensed matter (many materials
+fabricated in laboratories contain electrons that are strongly correlated and show exotic properties) to nuclear and
+particle physics.
+
+The proton, one of the particles inside an atom’s nucleus, is itself a strongly correlated bound state involving many
+quarks and gluons.
+Understanding its properties is an important research area in nuclear physics.
+The origin of mass and energy in the universe could be the result of strong correlations between fundamental quantum
+degrees.
+
+“Often we can write down the microscopic theory that describes a physical system.
+For example, we believe we know how quarks and gluons interact with each other to produce a proton.
+But then to go from there to calculate, for instance, the spin of the proton or its structure is non-trivial,” said
+Chandrasekharan.
+“Similarly, in a given material we have a good grasp of how electrons hop from one atom to another.
+However, from that theory to compute the conductivity of a strongly correlated material is very difficult.
+The final answer—that helps us understand things better—requires a lot of computation.
+Typically the computational cost grows exponentially with the number of interacting quantum degrees of freedom.”
+
+According to Chandrasekharan, the main challenge is to take this exponentially hard problem and convert it to something
+that scales as a polynomial and can be computed on a classical computer.
+“This step is often impossible for many strongly correlated quantum systems, due to the so-called
+[sign problem](http://en.wikipedia.org/wiki/Numerical_sign_problem) which arises due to quantum mechanics,” added
+Chandrasekharan.
+“Once the difficult sign problem is solved, we can use Monte Carlo calculations to obtain answers.
+Computing clusters like the OSG can be used at that stage.”
+
+Chandrasekharan has proposed an idea, called the [fermion bag approach](http://link.springer.com/article/10.1140%2Fepja%2Fi2013-13090-y),
+that has solved numerous sign problems that seemed unsolvable in systems containing fermions (electrons and quarks are
+examples of fermions).
+In order to understand a new mechanism for the origin of mass in the universe, Ayyar is specifically using the OSG to
+study an interacting theory of fermions using the fermion bag approach.
+
+<a href="/assets/images/Fermion-Bag-300x300.jpg"><img src="/assets/images/Fermion-Bag-300x300.jpg" alt="Fermion Bag" width="300" height="300" /></a>
+
+_Illustration of a fermion bag configuration. Image credit: Shailesh Chandrasekharan_
```

Comment:
Assuming the final pages are HTML 5, it might be nice to use the `figure` and `figcaption` elements here.